libs/tup-components/src/layout/Sidebar/Sidebar.module.css

```diff
@@ -1,5 +1,6 @@
 .root {
   overflow-y: auto;
+  height: 100%;
   flex-wrap: nowrap; /* Overwrite Bootstrap `_nav.scss` style for `.nav` */
```

Comment:
**Opportunity to delete code.** This style does not do anything now (cuz it is not a flex child) and that's good.

```suggestion
```